### PR TITLE
feat(NET-542) workflow to publish standalone docker builds

### PR DESCRIPTION
**Pull request by pwilson77**

### CODE REVIEW

File: .github/workflows/publish-qa-builds.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name: Publish QA Docker Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Docker image tag'
+        required: true
+        default: 'latest'
```
Comment (Aceix) [inline .github/workflows/publish-qa-builds.yml:9]: no default. user must enter a tag

File: .github/workflows/publish-qa-builds.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  release:
```
Comment (Aceix) [inline .github/workflows/publish-qa-builds.yml:10]: no release. should only be triggered manually (workflow_dispatch)

File: .github/workflows/publish-qa-builds.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        name: Set tag
+        run: |
+            if [[ -n "${{ github.event.inputs.tag }}" ]]; then
+                TAG=${{ github.event.inputs.tag }}
+            elif [[ "${{ github.ref_name }}" == 'master' ]]; then
+                TAG="latest"
+            else
+                TAG="${{ github.ref_name }}"
+            fi
+            echo "TAG=${TAG}" >> $GITHUB_ENV
```
Comment (Aceix) [inline .github/workflows/publish-qa-builds.yml:27]: this whole block can be changed to 
```
TAG=qa-testing-${{ github.event.inputs.tag }}
echo "TAG=${TAG}" >> $GITHUB_ENV
```

File: .github/workflows/publish-qa-builds.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+            context: .
+            file: ./Dockerfile.standalone
+            platforms: linux/amd64, linux/arm64, linux/arm/v7
+            push: true
+            tags: ${{ github.repository_owner }}/netmaker-ui:qa-testing${{ env.TAG }}
```
Comment (Aceix) [inline .github/workflows/publish-qa-builds.yml:51]: can use this (moved the `qa-testing` to the tag variable)

```
tags: ${{ github.repository_owner }}/netmaker-ui:${{ env.TAG }}
```

File: .github/workflows/publish-qa-builds.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
+name: Publish QA Docker Image
```
Comment (Aceix) [inline .github/workflows/publish-qa-builds.yml:1]: "Publish QA Standalone Docker Image"

**Review by Aceix — CHANGES_REQUESTED** [review]

File: .github/workflows/publish-qa-builds.yml (hunk 3 above)
Comment (pwilson77) [reply .github/workflows/publish-qa-builds.yml]: done

File: .github/workflows/publish-qa-builds.yml (hunk 2 above)
Comment (pwilson77) [reply .github/workflows/publish-qa-builds.yml]: removed

File: .github/workflows/publish-qa-builds.yml (hunk 1 above)
Comment (pwilson77) [reply .github/workflows/publish-qa-builds.yml]: removed

**Review by Aceix — APPROVED** [review]

**Comment by Aceix** [comment]
@pwilson77 raise another PR to `master` so you can test

**Pull request merged**